apps/mull-ui/src/app/components/event-members/event-members.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import './event-members.scss';
+
+export interface EventMembersProps {
+  profilePictures: string[];
+}
+
+export const EventMembers = ({profilePictures}: EventMembersProps) => {
+  const profileImages = profilePictures.map((image, index) => <img className="profile-picture" key={index} src={image} alt={`participant_${index+1}`} />);
```

Comment:
One problem with your styling is that it isn't very flexible, it expects there to be 3 images. If there are fewer than 3, the containers end up being larger than necessary. If there's more than 3, the 4th image and onward are still rendered, but hidden behind the 3rd one.

I would have tried writing the styling dynamically in javascript, however, I'm not sure if everyone would agree that that's a good idea (@r-saba let me know what you think). If you're interested in trying to implement a more flexible solution (dynamic width, dynamic number of pictures, etc.), let me know because I already had something in mind and can help you out.

However, if you want to keep things simple, please at least trim the list of profile images that are shown to the first three given in `profilePictures`, since that's the way you've hardcoded your styling be. that way we don't render hidden images unnecessarily.

Reply:
No preference for me, whatever you guys decide. Dynamic or not we should limit the array for sure like @TheGreatMarkus pointed out.

Reply:
Will do the array slicing and will attempt at dynamic styling. If I can't, when I begin, implement the dynamic part in an hour or two, we'll merge this and I'll leave the dynamic implementation to you @TheGreatMarkus. Is that okay?

Reply:
Sure I'd be down to implement it. You want me to merge the dynamic implementation as a separate pr?

Reply:
Sure, but can I just try something first very quick?